dpapi.go

```diff
@@ -61,12 +65,25 @@ func encrypt(secret string, cf cryptProtect) (string, error) {
 
 // EncryptBytes encrypts a byte array and returns a byte array
 func EncryptBytes(data []byte) ([]byte, error) {
-	return encryptBytes(data, cryptProtectUIForbidden)
+	return encryptBytes(data, []byte{}, cryptProtectUIForbidden)
```

Comment:
```suggestion
	return encryptBytes(data, nil, cryptProtectUIForbidden)
```

No need to allocate an empty `[]byte` here. Checking the `len` of a `nil` `[]byte` is also 0.